src/CoreText/CTFontManager.cs

```diff
@@ -686,18 +771,27 @@ public static void RegisterFonts (string[] assetNames, CFBundle bundle, CTFontMa
 			}
 		}
 
-#if !NET
-		[NoWatch, NoTV, NoMac, iOS (13,0)]
+#if NET
+		// [SupportedOSPlatform ("ios13.0")] - Not valid on delegate declaration
+		// [UnsupportedOSPlatform ("tvos")]
+		// [UnsupportedOSPlatform ("macos")]
```

Comment:
??? And this commented out attrs?

Reply:
fine to keep in the case that in the future they become available

Reply:
@mandel-macaque this is a manual task to comment out the attributes for the delegates

Reply:
Filed as https://github.com/dotnet/runtime/issues/64384